scarf/readers.py

```diff
@@ -34,28 +41,38 @@ def get_file_handle(fn: str) -> IO:
         else:
             return open(fn, 'r')
     except (OSError, IOError, FileNotFoundError):
-        raise FileNotFoundError("ERROR: FILE NOT FOUND: %s" % fn)
```

Comment:
Shouldn't this be `fn`

Reply:
oh yes, this must have just been an accidental deletion of a character, sorry